src/runtime/virtcontainers/kata_agent.go

```diff
@@ -68,8 +68,10 @@ const (
 	kernelParamDebugConsole           = "agent.debug_console"
 	kernelParamDebugConsoleVPort      = "agent.debug_console_vport"
 	kernelParamDebugConsoleVPortValue = "1026"
-)
 
+	// Restricted permission for shared directory managed by virtiofs
+	sharedDirMode = os.FileMode(0750) | os.ModeDir
```

Comment:
@c3d Shouldnt this be `os.FileMode(0700)` instead?

Reply:
I was pretty sure it was 0700 when I reviewed it for the first time.
It looks like some unintentional change, @c3d.

Reply:
Adding the `do-not-merge` till its solved.  Please, feel free to remove it when this is addressed.